csharp/Platform.IO/ConsoleCancellation.cs

```diff
@@ -64,6 +101,18 @@ protected override void Dispose(bool manual, bool wasDisposed)
             }
         }
 
+        /// <summary>
+        /// <para>Requests cancellation if cancellation was not requested.</para>
+        /// <para>Запрашивает отмену если отмена не запрашивалась.</para>
+        /// </summary>
+        /// <param name="sender">
+        /// <para>The event source.</para>
+        /// <para>Источник события.</para>
+        /// </param>
+        /// <param name="e">
+        /// <para>The <see cref="ConsoleCancelEventArgs"/> class instance that provides data for the CancelKeyPress event.</para>
+        /// <para>Экземпляр класса <see cref="ConsoleCancelEventArgs"/> предоставляющий данные для события CancelKeyPress.</para>
+        /// </param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
We do not comment private members of the class. This comment should be removed.

Reply:
Where can I see if this parameter is private or not?

Reply:
https://github.com/linksplatform/IO/pull/59/files/4c7afc23b935d9b5111cfcb06099d7fa524a964d#diff-c98a6da906f1c2555560c209d31ba44f61c94453d1686f12f05e6a96281b955fR117 - here for example.
You can read more about this here: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers

Reply:
`private` members are usually implementation details. That means that they are subject to frequent change. `public` and `protected` members of the class are API available to users of the library. That means that such member will never change, or will change slowly (usually with the next major release according to [semver](https://semver.org/))

Reply:
It is my carelessness. I thought you were talkin about parameter :D